src/main/java/seedu/guest/logic/parser/AddCommandParser.java

```diff
@@ -49,8 +50,9 @@ public AddCommand parse(String args) throws ParseException {
                 .parseNumberOfGuests(argMultimap.getValue(PREFIX_NUMBER_OF_GUESTS).get());
         IsRoomClean isRoomClean = ParserUtil
                 .parseIsRoomClean(argMultimap.getValue(PREFIX_IS_ROOM_CLEAN).get());
+        Bill bill = new Bill();
```

Comment:
If I understand, the prefix is to be added in the later stages right? If so, feel free to ignore this comment!

Reply:
There will be no prefix here as the bill will be updated using its own command, and not the add or edit commands